rollup.config.js

```diff
@@ -1,14 +1,21 @@
 import babel from 'rollup-plugin-babel';
+import alias from 'rollup-plugin-alias';
+
+const {targets} = require('./webpack.common');
 
 process.env.NODE_ENV = 'esm';
 
-export default {
-    input: 'katex.js',
+export default targets.map(({name, entry}) => ({
+    input: entry.replace('.webpack', ''),
     output: {
-        file: 'dist/katex.mjs',
+        file: `dist/${name}.mjs`,
         format: 'es',
     },
     plugins: [
         babel({runtimeHelpers: true}),
+        alias({
+            katex: '../katex.mjs',
```

Comment:
Does this require `katex` to appear first in `targets`?  Also, where is this used?

Reply:
@kevinbarabash It transforms `import katex from ‘katex’` to `import katex from ‘../katex.mjs’` as browsers cannot resolve a module by its name by spec.

Reply:
After I asked the question I ended up having to so something similar for another project.